src/util/Embeds/LobbyEmbed.ts

```diff
@@ -0,0 +1,62 @@
+import {
+    ActionRowBuilder,
+    ButtonBuilder,
+    ButtonStyle,
+    EmbedBuilder,
+} from "discord.js";
+import PlayerManager from "../Player/PlayerManager.js";
+
+export default class LobbyEmbed {
+    startTime: number;
+    players: PlayerManager;
+
+    constructor(players: PlayerManager) {
+        this.startTime = Math.floor(Date.now() / 1000 + 120);
+        this.players = players;
+    }
+
+    CreateEmbed(disableButtons: boolean = false) {
```

Comment:
Rename to "CreateMessagePayload" as it does not only return Embeds, but also returns components.
These are also both of the required components in MessagePayload to send, update, or edit an interaction's message.